### PR TITLE
desktop UI: steps, files confirmation and topbar

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: apps/desktop/src/app/main-layout/topbar.component.ts [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+    private router: Router,
+    private user: UserService,
+    private sync: SyncService,
+    private sdk: SDKService,
+  ) {}
+
+  menuOpen = false;
+
+  account = of(this.sync.getAccount()).pipe(
+    switchMap((account) => this.sdk.setCurrentAccount(account)),
```
Comment (ebrehault) [inline apps/desktop/src/app/main-layout/topbar.component.ts:27]: Why do we need to do that here?
It is already done in the SyncService constructor.
(I mean, maybe we need to repeat the `setCurrentAccount` for some reason, but if so, that's fine, even though it is a bit weird. Maybe at some point we will need a `getCurrentAccount` in SDKService)

File: apps/desktop/src/app/upload/steps/steps.component.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'da-steps',
+  templateUrl: './steps.component.html',
+  styleUrls: ['./steps.component.scss']
```
Comment (ebrehault) [inline apps/desktop/src/app/upload/steps/steps.component.ts:6]: OnPush

**Review by ebrehault — APPROVED** [review]

File: apps/desktop/src/app/upload/steps/steps.component.ts (hunk 2 above)
Comment (operramon) [reply apps/desktop/src/app/upload/steps/steps.component.ts]: fixed

File: apps/desktop/src/app/main-layout/topbar.component.ts (hunk 1 above)
Comment (operramon) [reply apps/desktop/src/app/main-layout/topbar.component.ts]: fixed

**Pull request merged**